esphome/components/scd30/sensor.py

```diff
@@ -79,6 +90,13 @@ async def to_code(config):
     if CONF_ALTITUDE_COMPENSATION in config:
         cg.add(var.set_altitude_compensation(config[CONF_ALTITUDE_COMPENSATION]))
 
+    if CONF_FORCED_CALIBRATION_BASELINE in config:
+        cg.add(
+            var.set_forced_calibration_baseline(
+                config[CONF_FORCED_CALIBRATION_BASELINE]
+            )
+        )
+
```

Comment:
This can then be changed
```suggestion
    cg.add(
        var.set_forced_calibration_baseline(
            config[CONF_FORCED_CALIBRATION_BASELINE]
        )
    )

```